src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -224,8 +232,10 @@ public VerifyMatchingService getVerifyMatchingService(
                     new DateTimeComparator(Duration.ZERO),
                     assertionDecrypter,
                     configuration.getCountry().getHubConnectorEntityId()
-                )
-            ));
+                ),
+                new EidasMatchingRequestToLMSRequestTransform(userIdHashFactory),
```

Comment:
The naming is a bit inconsistent? Shouldn't it be a "transformer" because the others are? Also we have been using MS (short for matching service) instead of LMS.